corehq/apps/app_manager/models.py

```diff
@@ -1316,6 +1316,28 @@ def _get_active_actions(self, types):
                 actions[action_type] = a
         return actions
 
+    @memoized
+    def get_action_type(self):
+        if self.get_subcase_types():
+            return 'open'
+        elif self.requires_case():
+            return 'update'
+        else:
+            return 'none'
+
+    @memoized
+    def get_icon_help_text(self):
+        subcases = self.get_subcase_types()
+        messages = []
+
+        if subcases:
+            messages.append(_('This form opens cases for {}').format(', '.join(subcases)))
+
+        if self.requires_case():
+            messages.append(('This form updates the {} case').format(self.get_module().case_type))
```

Comment:
I think it's not unusual to use "case" for the case type, which makes this `This form updates the case case.` How about `This form will update a {}`?